jaxrs/src/main/java/se/fortnox/reactivewizard/ExceptionHandler.java

```diff
@@ -21,15 +21,18 @@
 import javax.ws.rs.core.MediaType;
 import java.nio.channels.ClosedChannelException;
 import java.nio.file.FileSystemException;
+import java.util.HashSet;
 import java.util.List;
+import java.util.Set;
 
 import static se.fortnox.reactivewizard.jaxrs.RequestLogger.getHeaderValueOrRedact;
 
 /**
  * Handles exceptions and writes errors to the response and the log.
  */
 public class ExceptionHandler {
```

Comment:
Just a thought to make this easier to use; Seems to me like the list of sensitive headers would fit nicely in `/server/src/main/java/se/fortnox/reactivewizard/server/ServerConfig.java`, perhaps it would be possible to make a subclass of `ExceptionHandler` in the `server` module and override `getHeaderValueOrRedact` in tht subclass, taking the `ServerConfig` as a constructor parameter?